go/nbs/table_reader.go

```diff
@@ -314,6 +305,10 @@ func (tr tableReader) findOffsets(reqs []getRecord) (ors offsetRecSlice, remaini
 	// Iterate over |reqs| and |tr.prefixes| (both sorted by address) and build the set
 	// of table locations which must be read in order to satisfy |reqs|.
 	for i, req := range reqs {
+		if req.found {
```

Comment:
There's a race here that we discussed offline. We should be OK for the immediate future, but will fix ASAP